packages/frontend-2/components/server-management/DeleteInvitationDialog.vue

```diff
@@ -0,0 +1,46 @@
+<template>
+  <LayoutDialog v-model:open="isOpen" max-width="sm" :title="title" :buttons="buttons">
+    <div class="flex flex-col gap-6 text-sm text-foreground">
+      <p>Are you sure you want to delete the selected invitation?</p>
+      <div v-if="invite" class="flex flex-col gap-2">
+        <div class="flex gap-1">
+          <div class="w-20">Email:</div>
+          <strong>{{ invite?.email }}</strong>
```

Comment:
Redundant (& misleading) optional chaining operator, its already established that `invite` is defined on line 5